packages/parallax/demo/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
maybe these files could be in `packages/parallax/test/*` to match the signature of the rest of the repo. Probably a good idea to add a little readme explaining this is for e2e tests too, what do you think?

Reply:
Yeah, both of those make sense to me. It isn't really a demo in that we don't want anyone messing with it unless they are writing new tests. And a `README` will double down on making that clear.

Reply:
exactly. ⭐

Reply:
Changed name and added `README` explaining it's for testing and how to generate new snapshots should someone contribute tests that alter the DOM.